draft-ietf-acme-acme.md

```diff
@@ -1885,21 +1886,17 @@ action the client should take:
 * "invalid": The certificate will not be issued.  Consider this order process
   abandoned.
 
-* "pending": The server does not believe that the client has fulfilled the
-  requirements.  Check the "authorizations" array for entries that are still
-  pending.
-
-* "ready": The server agrees that the requirements have been
-  fulfilled, and is awaiting finalization.  Submit a finalization
-  request.
-
 * "processing": The certificate is being issued. Send a POST-as-GET request after the
   time given in the Retry-After header field of the response, if
   any.
 
 * "valid": The server has issued the certificate and provisioned its URL to the
   "certificate" field of the order.  Download the certificate.
 
+* "pending": This status should not be returned after a successful finalize request.
```

Comment:
L1878-L1879 references these status definitions when an "orderNotReady" failure is returned, so I believe fully defining each of the statuses is clearer.

**Note "The status of the order will indicate what action the client should take" is repeated twice within three lines so it may be helpful to additionally reword it, but I didn't come up with a quick fix.

Reply:
Good point, thanks. Reverted this portion of the diff.

Reply:
@jsha Can you also update the PR desc to remove mention of the removed diff?

Reply:
Done.